include/avif/avif.h

```diff
@@ -1524,6 +1547,22 @@ typedef enum avifAddImageFlag
 } avifAddImageFlag;
 typedef uint32_t avifAddImageFlags;
 
+// Arguments passed to avifEncoderCustomEncodeImageFunc by the avifEncoder instance.
+typedef struct avifEncoderCustomEncodeImageArgs
+{
+    // Encoding settings requested by the avifEncoder instance for the current AV1 coded image item or sequence.
+    avifAddImageFlags addImageFlags;
+    int quantizer;    // AV1 quality setting in range [AVIF_QUANTIZER_BEST_QUALITY:AVIF_QUANTIZER_WORST_QUALITY].
+    int tileRowsLog2; // Logarithm in base 2 of the number of AV1 tile rows.
+    int tileColsLog2; // Logarithm in base 2 of the number of AV1 tile columns.
```

Comment:
You would need to have the coordinates of the current tile being encoded, no?

Reply:
Added `gridRow`/`gridColumn` in `avifEncoderCustomEncodeImageItem`.